browser/resources/settings/email_aliases_page/content/email_aliases_list.tsx

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+import { Alias, MappingService, MAX_ALIASES, ViewState } from './types'
+import Card from './styles/Card'
+import Col from './styles/Col'
+import Row from './styles/Row'
+import styled from 'styled-components'
+import Icon from '@brave/leo/react/icon'
+import { color } from '@brave/leo/tokens/css/variables'
+import Tooltip from '@brave/leo/react/tooltip'
+import { getLocale } from '$web-common/locale'
+import formatMessage from '$web-common/formatMessage'
+import Button from '@brave/leo/react/button'
+import ButtonMenu from '@brave/leo/react/buttonMenu'
+
+const AliasItemRow = styled(Row)`
+  margin: 0px;
+  font-size: 125%;
+  padding: 18px 0px 18px 25px;
+  border-top: ${color.divider.subtle} 1px solid;
+  justify-content: space-between;
+`
+
+const AliasAnnotation = styled.div`
+  font-size: 80%;
+  font-weight: 400;
+  padding-top: 0.25em;
+  color: ${color.neutralVariant[50]};
+`
+
+const AliasControls = styled(Row)`
+  height: 1.5em;
```

Comment:
we shouldn't need to set an explicit height here - it should come from the intrinsic height of the content.

Also, setting an em height is probably not correct - icons are given a pixel size so they won't necessarily fit in this container.

Reply:
fixed